Modules.Conan/FindCatch.cmake

```diff
@@ -0,0 +1,12 @@
+find_package(Catch2 REQUIRED CONFIG)

```

Comment:
Do we use Catch or Catch2?

Reply:
@malytomas used Catch2 for Windows build, since Catch is not available in Conan.

Reply:
FindCatch.cmake changed to FindCatch2.cmake